spec.md

```diff
@@ -445,9 +445,99 @@ Resource properties:
 * `name` - optional
 * Links to any metadata objects controlled by the DataPub server
 * Links to any metadata objects modifiable by the resource's controller
-### Resource Operations
 
-#### Resource HTTP API
+### Create Resource Operation
+
+A resource is created by making a Create Resource request to a given [=space=],
+and (optionally) a [=collection=]. If a collection is not explicitly specified,
+the resource is created in that space's [=default collection=].
+
+Using the HTTP API, a resource can be created either by making a POST request
+to a space and collection, or by making a PUT request to a specific resource
+(see the Update Resource operation).
+
+#### (HTTP API) POST `/spaces/{space_id}/{collection/*}`
```

Comment:
Idea: what if we dont even document these non-idempotent POST things?

We leave them as an accessory spec for non-idempotent but good-dx features?

Reply:
(or appendix)

Reply:
(as mentioned during our convo yesterday, by requiring CIDs, we make POST idempotent. I'll make sure to clarify that in the spec)